### ISSUE TITLE
Selenide 7.4.2 has broken selectOptionContainingText and selectOptionByValue methods

### ISSUE THREAD

**Issue by farious2009** [selenide/selenide#2843]
Hi there,
I am writing here, because it seems like this update broke some of Selenide methods which works with select option. 

Currently I am aware of these 2:

```
void selectOptionContainingText(String var1, String... var2);
void selectOptionByValue(String var1, String... var2);
```

**History in a nutshell:**

I wanted to bump up selenide to get rid of choosing engine pop up message, so upgraded from

```
<selenide.version>7.2.1</selenide.version>
<selenium.version>4.18.1</selenium.version>
```

to

```
<selenide.version>7.4.2</selenide.version>
<selenium.version>4.23.0</selenium.version>	
```

and those 2 stopped working. 

"Stopped working" means here that selenide chooses the option from select drop-down menu, but then nothing happens next where previously depended on chosen option fields should have appeared or items to be sorted upon choosing a respective ASC or DESC or something else. I even checked click on then chosen element and it clicked successfully somewhere in a background. So, because, of course, no field appeared called by ajax, our tests started failing.

It just so happened that we had at +- around that time updates on the frontend (FE) part, so we thought that the non-working select choice was related to that FE part, and we wrote a temporal "workaround" (utilities) to find needed element among ElementCollection and click on it and it helped out.
```
        select.$$("option").shouldHave(sizeGreaterThan(1))
                .asDynamicIterable()
                .stream()
                .filter(filterPredicate)
                .findFirst()
                .orElseThrow()
                .click();
```

___		
Then I had changed frontend part to a previous version before FE changes and those 2 selenide methods still were not working, so I downgraded to 
```
<selenide.version>7.4.1</selenide.version>
<selenium.version>4.23.0</selenium.version>
```

and methods have started working again.	
		
Maybe "Select" change within 7.4.2 broke something as 7.4.3 does not working for us  either ? 

I cannot add code here from prod, but we do check simple sorting of the items on the page by either value or text.

Kind regards

**Comment by farious2009** [comment]
![sorting](https://github.com/user-attachments/assets/93b1dc93-ccaa-474c-91bb-21919de5f96b)

**Comment by farious2009** [comment]
Used to work before the update
sortingSelectElement.shouldBe(visible, enabled).selectOptionByValue(value);

**Comment by asolntsev** [comment]
@farious2009 I guess this problem was already described here: https://github.com/selenide/selenide/issues/2832
And fixed here: https://github.com/selenide/selenide/pull/2835

Will be released soon as Selenide 7.5.0

**Comment by farious2009** [comment]
Thank you!
